### PR TITLE
Update WebBluetooth webidl files

### PR DESCRIPTION
**Pull request by zakorgy**
Update the WebBluetooth related webidl files with the `[SameObject]` extended attribute, and unify the indentation in these files.

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors

- [x] There are tests for these changes

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/15367)

### CODE REVIEW

File: components/script/dom/webidls/Navigator.webidl [hunk 1]
```diff
@@ -28,7 +28,7 @@ interface NavigatorID {
 
 // https://webbluetoothcg.github.io/web-bluetooth/#navigator-extensions
 partial interface Navigator {
-  readonly attribute Bluetooth bluetooth;
+  [SameObject, Pref="dom.bluetooth.enabled"] readonly attribute Bluetooth bluetooth;
```
Comment (nox) [inline components/script/dom/webidls/Navigator.webidl]: What is this? This change isn't described in the commit message.

Comment (zakorgy) [inline components/script/dom/webidls/Navigator.webidl]: @nox you are right. I forgot to mention, I added the `dom.bluetooth.enabled` pref here. Without this pref here, other Bluetooth interfaces can be accessed via this attribute without passing `--pref="dom.bluetooth.enabled"` when running servo, even though those interfaces have this restriction also and should not be accessed. As far as I know the purpose of this `dom.bluetooth.enabled` pref (which is  set to `false`) is to disable the webbluetooth features by default, but without this addition, it does not.

Comment (nox) [inline components/script/dom/webidls/Navigator.webidl]: Thanks for the explanation.

**Pull request merged**